config/rbac/role.yaml

```diff
@@ -93,3 +93,25 @@ rules:
   - get
   - patch
   - update
+- apiGroups:
+  - ipam.cluster.x-k8s.io
+  resources:
+  - ipaddressclaims
+  - ipaddressclaims/status
+  verbs:
```

Comment:
Nope, I'll remove it from the markers, it will be needed for https://github.com/kubernetes-sigs/cluster-api-provider-openstack/pull/1763 but it shouldn't be in the openstackmachine markers